src/device/pf_diag.c

```diff
@@ -311,6 +328,16 @@ int pf_diag_add (
             }
             else
             {
+               LOG_DEBUG (
+                  PNET_LOG,
```

Comment:
Maybe a separate define for diag/alarms?

Reply:
Used PF_ALARM_LOG instead.
Done